src/entities/trainer/ui/trainers-card/trainer-card.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames/bind';
+import { Trainer } from '../../types';
+import { Image } from '@/shared/ui/image';
+
+import styles from './trainer-card.module.scss';
+
+const cx = classNames.bind(styles);
+
+type TrainerProps = Trainer;
+
+export const TrainerCard = ({ name, bio, role, photo }: TrainerProps) => {
+  return (
+    <article className={cx('trainer-card')}>
+      <div className={cx('trainer-card__picture')}>
+        <Image src={photo} alt={`${name} ${role}`} />
+      </div>
+      <div className={cx('trainer-card__right')}>
```

Comment:
```suggestion
      <div className={cx('trainer-card__text/description')}>
```

Reply:
fixed in https://github.com/rolling-scopes/site/pull/492/commits/0a1775b5be67bfe9689d7da53211f49cdd03e85d